reference/configuration/doctrine.rst

```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
 ``default_connection`` parameter.
 
 Each connection is also accessible via the ``doctrine.dbal.[name]_connection``
-service where ``[name]`` is the name of the connection.
+service where ``[name]`` is the name of the connection. Or directly in the controller with 
+getConnection method and the name of the connection.
+
+.. code-block:: php
```

Comment:
You have an error `"Please do not use ".. code-block:: php", use "::" instead."`. See the example how to insert php code in other files, for example here: https://raw.githubusercontent.com/symfony/symfony-docs/master/reference/configuration/kernel.rst

Reply:
Please remove this